src/workloads/query/TimeseriesCount.yml

```diff
@@ -0,0 +1,126 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  The queries in this workload exercise group stage that uses the $count accumulator and the $count 
+  aggregation stage. On FCV greater than or equal to 7.1 $group using $count is optimized to remove 
+  the $unpack stage.
+
+Keywords:
+- timeseries
+- aggregate
+- group
+
+GlobalDefaults:
+  Database: &database test
+  Collection: &collection Collection0
+  MaxPhases: &maxPhases 10
+  repeat: &repeat 100
+
+Actors:
+# Clear any pre-existing collection state.
```

Comment:
Is this required for local testing? Any patch/waterfall run should start out with an empty DB.